custom_components/waste_collection_schedule/waste_collection_schedule/source/ccc_govt_nz.py

```diff
@@ -3,11 +3,37 @@
 import requests
 from waste_collection_schedule import Collection
 
+# Updated to work around SSL UNSAFE_LEGACY_RENEGOTIATION_DISABLED error using method discussed in
```

Comment:
This is the same issue that we already had with another source, right? Could you move the common code into a file in the services folder and import it from there?

I think I've seen another source with the same issue: bielefeld_de
Could you also fix this one using the common code?

And finally: Could you please refactor the original source with this issue so that is also uses the common code?

Thanks!

Reply:
services folder now contains a SSLError.py file containing the common code